src/view/dataElements/xdmObject/helpers/mainViewState.js

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2021 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import * as STATUS from "../constants/mainViewStatus";
+
+export const ACTION_TYPES = {
+  DEFAULT_SANDBOX_AND_SCHEMA_LOADED: "defaultSandboxAndSchemaLoaded",
+  SELECTED_SANDBOX_CHANGED: "selectedSandboxChanged",
+  SCHEMA_LOADED: "schemaLoaded",
+  SELECTED_SCHEMA_META_CHANGED: "selectedSchemaMetaChanged",
+  SELECTED_NODE_ID_CHANGED: "selectedNodeIdChanged",
+  UPDATE_SHOW_EDITOR_NOT_READY_VALIDATION_ERROR:
+    "updateShowEditorNotReadyValidationError"
+};
```

Comment:
I've [debated proper capitalization methods](https://adobedx.slack.com/archives/GPKKPT0Q1/p1622237500146300). It seems we've been inconsistent in these cases where we basically have an enum object. Airbnb [has a stance](https://github.com/airbnb/javascript/#naming--uppercase) (`ACTION_TYPES` uppercase, `defaultSandboxAndSchemaLoaded` lowercase), which I'd be cool with, but it gets interesting when we have this: `import * as STATUS from "../constants/mainViewStatus";` The constants are named exports inside `mainViewStatus.js`, so they're uppercased...meaning we end up with `STATUS.SOMETHING` rather than `STATUS.something`.

Reply:
I think I like that option less than just keeping them all caps.